bs/src/tablecloth.ml

```diff
@@ -441,6 +441,13 @@ module Result = struct
   let map (f : 'ok -> 'value) (r : ('err, 'ok) t) : ('err, 'value) t =
     Belt.Result.map r f
 
+  let fromOption left ma = 
```

Comment:
Maybe `left` could be renamed (errorMessage?) and optional with a default error message being provided. 

Then the usage would be:
```
Result.fromOption None
```
or
```
Result.fromOption ~errorMessage:"More specific message" None
```

Reply:
I agree `left` is too vague.

The problem I see with `errorMessage` is that `message` implies it's a `string` when it should really be `'a`.

I'll change it to `error` so we can do `Result.fromOption ~error: "error message" None`.

Also since it's `'a` instead of `string`, I don't think it makes sense to provide a default value and narrow the types unnecessarily.